src/classes/CRLP_Batch_Base_NonSkew.cls

```diff
@@ -75,6 +87,24 @@ public abstract class CRLP_Batch_Base_NonSkew extends CRLP_Batch_Base {
         super(jobType, jobFilter);
     }
 
+    /*******************************************************************************************************
+     * @description Constructs the class - Rollup is for a specific set of Summary Object Id's
+     * @param jobType Defines the type of rollup job
+     * @param recordIds Specific set of parent record id's to roll up to
+     */
+    public CRLP_Batch_Base_NonSkew(CRLP_RollupProcessingOptions.RollupType jobType, Id lastIdProcessedForChunking) {
```

Comment:
Both var names could be `lastIdProcessed`.

Reply:
I like it - updated!